ignite/cmd/scaffold_single.go

```diff
@@ -9,8 +9,11 @@ import (
 // NewScaffoldSingle returns a new command to scaffold a singleton.
 func NewScaffoldSingle() *cobra.Command {
 	c := &cobra.Command{
-		Use:     "single NAME [field]...",
-		Short:   "CRUD for data stored in a single location",
+		Use:   "single NAME [field:type]...",
+		Short: "CRUD for data stored in a single location",
+		Long: `CRUD for data stored in a single location.
+		
+		For detailed type information use ignite scaffold type --help.`,
 		Args:    cobra.MinimumNArgs(1),
```

Comment:
Can you also add an example to the `Example` parameter?

Reply:
Done, also do you think we need to add `Example` for other scaffold commands?

Reply:
I think we can only cover these query commands in this PR and others later, wdyt?

Reply:
Sounds good to me